CMakePresets.json

```diff
@@ -0,0 +1,307 @@
+{
+    "version": 3,
+    "configurePresets": [
+        {
+            "hidden": true,
+            "name": "linux",
+            "condition": {
+                "type": "equals",
+                "lhs": "${hostSystemName}",
+                "rhs": "Linux"
+            }
+        },
+        {
+            "hidden": true,
+            "name": "ci-base",
+            "displayName": "CI configuration",
+            "description": "Sets Ninja generator, build and install directory",
+            "generator": "Ninja",
+            "binaryDir": "${sourceDir}/out/build/${presetName}",
+            "installDir": "${sourceDir}/out/install/${presetName}",
+            "cacheVariables": {
+                "CMAKE_BUILD_TYPE": "Release",
+                "BLA_VENDOR": "Generic"
```

Comment:
*This is a new-to-simbody configuration option [`BLA_VENDOR=Generic`](https://cmake.org/cmake/help/latest/module/FindBLAS.html#blas-lapack-vendors) added to the base preset (i.e. applies to all presets).

This setting should be functionally irrelevant for CI builds, which only have one BLAS/LAPACK installed, but should be more universal for user's with multiple installed BLAS/LAPACK's. Any manually specified cache variables (e.g. on the command line or via GUI) will override preset settings, so users who wish to use a specific BLAS/LAPACK (either by picking a different/specific BLA_VENDOR, or by setting BUILD_USING_OTHER_LAPACK). Defaulting to Generic BLAS/LAPACK allows more vendor-agnostic linking, such as allowing post-build swapping of the default BLAS/LAPACK's using update-alternatives, which I have tested does work.)